Lib/tracemalloc.py

```diff
@@ -226,7 +226,7 @@ def __str__(self):
         return str(self[0])
 
     def __repr__(self):
-        s = "<Traceback %r" % tuple(self)
+        s = f"<Traceback {tuple(self)}"
```

Comment:
I'm confused about what this changes?

Reply:
`"%r" % (1,2,3)` raises: "TypeError: not all arguments converted during string formatting".
This was changed (apparently unintentionally) in https://github.com/python/cpython/commit/8d59eb1b66c51b2b918da9881c57d07d08df43b7#diff-cbaacb7ac396956c9595b4899ef2ab31b02b264099e02eba56fbe2f5f2e441c0L224-R229.

Reply:
nice catch